claimstore/modules/claims/models.py

```diff
@@ -274,3 +300,155 @@ class Predicate(db.Model):
     def __repr__(self):
         """Printable version of the Predicate object."""
         return '<Predicate {}'.format(self.name)
+
+
+class EquivalentIdentifier(db.Model):
+
+    """Model that defines equivalent identifiers.
+
+    A given tuple (IdentifierType1, value1), e.g. (DOI, 1234) will have
+    associated a unique id (`eqid`) that will be shared by the other tuple,
+    e.g. (IdentifierType2, value2), in the relationship. If new equality claims
```

Comment:
You could add to the docstring: "... in the relationship, in case the two IDs are equivalent (e.g. related via `is_same_as` predicate).  If new equality claims..."